src/api/actions/uploadInBackground.ts

```diff
@@ -0,0 +1,147 @@
+import { checkBulkUploadJob } from "./checkBulkUploadJob";
+import { startBulkUpload } from "./startBulkUpload";
+import {
+  JobIdToUrl,
+  UploadJobStatus,
+  UrlToIPFS,
+  UrlToJobId,
+} from "../../types";
+import * as fs from "fs";
+
+const URLS_PER_CHUNK_START = 100;
+const URLS_PER_CHUNK_CHECK = 1000;
+const ERROR_FILEPATH = "error.json";
+const FAILURE_FILEPATH = "failed.json";
```

Comment:
What's the different between failure and start? Can this be explained via function header?